diff --git a/bin/console b/bin/console
index 855c82e4..68d1b525 100755
--- a/bin/console
+++ b/bin/console
@@ -7,7 +7,7 @@ use Symfony\Component\Debug\Debug;
 
 // if you don't want to setup permissions the proper way, just uncomment the following PHP line
 // read http://symfony.com/doc/current/book/installation.html#configuration-and-setup for more information
-umask(0000);
+//umask(0000);
 
 set_time_limit(0);
 
@@ -18,6 +18,7 @@ $env = $input->getParameterOption(['--env', '-e'], getenv('SYMFONY_ENV') ?: 'dev
 $debug = getenv('SYMFONY_DEBUG') !== '0' && !$input->hasParameterOption(['--no-debug', '']) && $env !== 'prod';
 
 if ($debug) {
+    umask(0000);
     Debug::enable();
 }
 
diff --git a/web/app.php b/web/app.php
index a1d2bdf4..5d2e252d 100644
--- a/web/app.php
+++ b/web/app.php
@@ -5,7 +5,7 @@ use Symfony\Component\HttpFoundation\Request;
 // If you don't want to setup permissions the proper way, just uncomment the following PHP line
 // read http://symfony.com/doc/current/book/installation.html#checking-symfony-application-configuration-and-setup
 // for more information
-umask(0000);
+//umask(0000);
 
 require __DIR__.'/../vendor/autoload.php';
 if (PHP_VERSION_ID < 70000) {
