diff --git a/ansible/playbook.yml b/ansible/playbook.yml
index 9e2cfa3..a744aa3 100644
--- a/ansible/playbook.yml
+++ b/ansible/playbook.yml
@@ -5,6 +5,7 @@
     server_name: mootube.l
     symfony_root_dir: /var/www/project
     symfony_web_dir: "{{ symfony_root_dir }}/web"
+    symfony_var_dir: "{{ symfony_root_dir }}/var"
 
   tasks:
     - ping: ~
@@ -88,6 +89,9 @@
     - name: Install Composer's dependencies
       composer: working_dir="{{ symfony_root_dir }}" no_dev="no"
 
+    - name: Fix var directory permissions
+      file: path="{{ symfony_var_dir }}" state=directory mode=0777 recurse=yes
+
   handlers:
     - name: Restart Nginx
       become: yes
diff --git a/web/app.php b/web/app.php
index 4c2c465..737673b 100644
--- a/web/app.php
+++ b/web/app.php
@@ -2,6 +2,11 @@
 
 use Symfony\Component\HttpFoundation\Request;
 
+// If you don't want to setup permissions the proper way, just uncomment the following PHP line
+// read http://symfony.com/doc/current/book/installation.html#checking-symfony-application-configuration-and-setup
+// for more information
+umask(0000);
+
 /**
  * @var Composer\Autoload\ClassLoader
  */
diff --git a/web/app_dev.php b/web/app_dev.php
index 8456754..856f849 100644
--- a/web/app_dev.php
+++ b/web/app_dev.php
@@ -6,7 +6,7 @@ use Symfony\Component\Debug\Debug;
 // If you don't want to setup permissions the proper way, just uncomment the following PHP line
 // read http://symfony.com/doc/current/book/installation.html#checking-symfony-application-configuration-and-setup
 // for more information
-//umask(0000);
+umask(0000);
 
 // This check prevents access to debug front controllers that are deployed by accident to production servers.
 // Feel free to remove this, extend it, or make something more sophisticated.
