diff --git a/ansible/deploy.yml b/ansible/deploy.yml
index 08b20ed..1c0cc51 100644
--- a/ansible/deploy.yml
+++ b/ansible/deploy.yml
@@ -2,6 +2,7 @@
 - hosts: aws
 
   vars_files:
+    - ./vars/vault.yml
     - ./vars/vars.yml
 
   vars_prompt:
diff --git a/app/config/parameters.yml.dist b/app/config/parameters.yml.dist
index 3d64fe2..b7385fb 100644
--- a/app/config/parameters.yml.dist
+++ b/app/config/parameters.yml.dist
@@ -16,7 +16,7 @@ parameters:
     mailer_password:   ~
 
     # A secret key that's used to generate certain security-related tokens
-    secret:            ThisTokenIsNotSoSecretChangeIt
+    secret:            "{{ symfony_secret }}"
 
     redis_host:        localhost
-    loggly_token:      12345
+    loggly_token:      "{{ loggly_token }}"
