diff --git a/app/Resources/views/default/index.html.twig b/app/Resources/views/default/index.html.twig
index f72e942..3a76f32 100644
--- a/app/Resources/views/default/index.html.twig
+++ b/app/Resources/views/default/index.html.twig
@@ -22,7 +22,7 @@
         </div>
         <div class="video-box">
             <div class="video-box-header">
-                <h3>Over {{ totalVideoUploadsCount|number_format }} Bovine Fitness Routines</h3>
+                <h3>Over {{ totalVideoUploadsCount|number_format }} Bovine Fitness Routines!!!</h3>
                 <div>Over {{ totalVideoViewsCount|number_format }} Total Video Views</div>
             </div>
             {% if tags %}
