diff --git a/app/config/config.yml b/app/config/config.yml
index 2be6446c..c765f3f1 100644
--- a/app/config/config.yml
+++ b/app/config/config.yml
@@ -18,7 +18,7 @@ framework:
     form:            ~
     csrf_protection: ~
     validation:      { enable_annotations: true }
-    #serializer:      { enable_annotations: true }
+    serializer:      { enable_annotations: true }
     templating:
         engines: ['twig']
     default_locale:  "%locale%"
diff --git a/src/AppBundle/Controller/DefaultController.php b/src/AppBundle/Controller/DefaultController.php
index e77d6306..73c58fab 100644
--- a/src/AppBundle/Controller/DefaultController.php
+++ b/src/AppBundle/Controller/DefaultController.php
@@ -58,6 +58,17 @@ class DefaultController extends Controller
         return $this->render('default/about.html.twig');
     }
 
+    /**
+     * @Route("/api/videos")
+     */
+    public function videosApiAction()
+    {
+        $videos = $this->getVideoRepository()
+            ->findAll();
+
+        return $this->json(['videos' => $videos]);
+    }
+
     /**
      * @param Video[] $videos
      *
