diff --git a/src/AppBundle/Controller/DefaultController.php b/src/AppBundle/Controller/DefaultController.php
index 962fdb8..6148d98 100644
--- a/src/AppBundle/Controller/DefaultController.php
+++ b/src/AppBundle/Controller/DefaultController.php
@@ -58,6 +58,17 @@ class DefaultController extends Controller
     }
 
     /**
+     * @Route("/api/videos")
+     */
+    public function videosApiAction()
+    {
+        $videos = $this->getVideoRepository()
+            ->findAll();
+
+        return $this->json(['videos' => $videos]);
+    }
+
+    /**
      * @param Video[] $videos
      *
      * @return array
